java/arcs/android/devtools/DevToolsService.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.android.devtools
+
+import android.app.Service
+import android.content.Intent
+import android.os.IBinder
+import kotlinx.coroutines.CoroutineName
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.cancel
+
+/**
+ * Implementation of [Service] for devtools to enable a [DevWebSocket] connection to a remote
+ * device by exposing the [IDevToolsService] API.
+ */
+class DevToolsService : Service() {
+
+    private val coroutineContext = Dispatchers.IO + CoroutineName("DevtoolsService")
```

Comment:
@ianull suggestion is to use `Dispatchers.DEFAULT` for Android as thread count is core based rather than `64`.